example.diff

```diff
@@ -159,4 +159,29 @@ index e69de29..eee417f 100644
 --- a/dir1/file.txt
 +++ b/dir2/file.txt
 @@ -0,0 +1 @@
-+new text
\ No newline at end of file
++new text
+diff --git a/dir3/ocaml.txt b/dir3/ocaml.txt
+index 08ch6s5..r4jk867
+--- a/dir3/ocaml.txt
++++ b/dir3/ocaml.txt
+@@ -0,0 +1,2 @@
++OCaml is an industrial-strength functional programming language with an emphasis on expressiveness and safety.
++Learning OCaml is interesting.
+@@ -3,4 +3,4 @@
+-Want to begin?
++Take a tour at https://ocaml.org/docs/tour-of-ocaml
++Write your first OCaml program here - https://ocaml.org/docs/your-first-program
+diff --git a/dir3/outreachy.txt b/dir3/outreachy.txt
+index 0xchj65..e03c6s4
+--- a/dir3/outreachy.txt 
++++ b/dir3/outreachy.txt
+@@ -0,0 +1,2 @@
++Outreachy provides internships in open source and open science. 
++Outreachy provides internships to people subject to systemic bias and impacted by underrepresentation in the technical industry where they are living.
+@@ -5,6 +5,7 @@
```

Comment:
The hunks do not show any context (common lines). How did you generate it?

Reply:
Okay, it's hand written. Perhaps I should have it this way:


        -Outreachy provides internships in open source and open science. 
        +Outreachy provides internships to people subject to systemic bias and impacted by underrepresentation in the technical industry where they are living.

Any better ideas please?
@Julow

Reply:
What do you think of writing two files that would generate the wanted diffs with `git diff --no-index file1 file2` ?

Reply:
Thanks a lot for this, I have generated the diffs as recommended.

Ready for another review.
@Julow